x-pack/qa/kerberos-tests/src/test/java/org/elasticsearch/xpack/security/authc/kerberos/KerberosAuthenticationIT.java

```diff
@@ -90,13 +89,13 @@ public void setupRoleMapping() throws IOException {
         assertOK(response);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/56507")
-    public void testSuppressedOnJDK15() {
-        assumeTrue("AwaitsFix on JDK15", JavaVersion.current().compareTo(JavaVersion.parse("15")) < 0);
+    // Skip tests for jdk 8u262 https://github.com/elastic/elasticsearch/issues/56507
```

Comment:
I'd prefer this comment was more descriptive so that readers don't have to follow the URL.
```suggestion
    //  JDK 8u262 shipped with a NPE in Kerberos code, see https://github.com/elastic/elasticsearch/issues/56507
```